includes/editor-templates/panel.php

```diff
@@ -186,22 +186,22 @@
 		</div>
 	<# } #>
 
-	<?php if ( ! Utils::has_pro() ) : ?>
-	<# if ( elementData.upsale_data && elementData.upsale_data.description ) { #>
+	<# if ( elementData.upsale_data && elementData.upsale_data.condition ) { #>
 	<div class="elementor-nerd-box elementor-nerd-box--upsale">
 		<# if ( elementData.upsale_data.image ) { #>
-		<img class="elementor-nerd-box-icon" src="<?php Utils::print_unescaped_internal_string( '{{ elementData.upsale_data.image }}' ); ?>" loading="lazy" alt="<?php echo esc_attr__( 'Upgrade', 'elementor' ); ?>" />
```

Comment:
elementData.upsale_data.image is used by a filter in the hosting team to override images widget promotions for cloud hosting.

Reply:
No, my intention was to allow 3rd party addon developers to create their own promotions.

Reply:
We're aligned on adding more flexibility.

Reply:
@Barcov Template updated.